CHANGES

```diff
@@ -9,20 +9,24 @@
 
  Changes between 1.1.1o and 1.1.1p [xx XXX xxxx]
 
-  *)
+  *) When OpenSSL TLS client is connecting without any supported elliptic
+     curves and TLS-1.3 protocol disabled the connection will no longer fail
```

Comment:
protocol *is* disabled

Reply:
Fixed @mattcaswell assuming this is still OK.